app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
```

Comment:
- [ ] @ismayito Great work implementing the change requested, however, the post should belong to the author as the previous review had noted, kindly consider changing the belong_to relationship.
```suggestion
  belongs_to :author,   class_name: 'User'
```